app/data/models/customer.js

```diff
@@ -0,0 +1,13 @@
+module.exports = (sequelize, DataTypes) => {
+  const customer = sequelize.define('customer', {
+    customerId: { type: DataTypes.INTEGER, primaryKey: true, autoIncrement: true },
+    legacyId: DataTypes.STRING,
+    frn: DataTypes.STRING
```

Comment:
Should we store the legacy type like we do in `ffc-pay-enrichment`?

Or is the intention here to only support one type of legacyId?

If so, should we rename that field to match that type?

Reply:
The look up in TL is checking that both the legacyType and legacyId matches. LegacyId being the trader/vendor number passed in, and type being the type of file (i.e. DPS). Given the service is purely checking out the DPS service, I felt it made sense to drop the type as we'd expect all of them to match, and when pulling a cut of the TL data just copy over the ones with the matching type (where the type === 3). That would just leave us in the table with rows that have the legacyId we are looking for (that the file has) and the FRN we want that corresponds to said Id. As such I don't feel we need the legacy type as well

Reply:
Is type `3` trader or vendor number?

Reply:
you've got me questioning myself now.... I'll revisit once I've rechecked

Reply:
I agree with you though, let's just save the one we need.  But makes sense to name the one we need with a name that we know what it is, like `sbi`, `trader` or `vendor`.

Reply:
trader number. have amended the name to be 'traderId' rather than 'legacyId'